### DISCUSSION TITLE
Swizzling footer

### DISCUSSION THREAD

**Discussion by Karan-S-dev** [facebook/docusaurus#8022] [Q&A] [answered]
I have swizzled footer component, using "npm run swizzle @docusaurus/theme-classic Footer -- --eject"

I have done eject because I wanted to use this footer in specific files only.

Now after running this command, a folder is created. 
Ejected Footer from @docusaurus/theme-classic to
- "C:\Users\karans\Rapidx_Documentation\website\src\theme\Footer\index.js"

Now how to proceed, I tried to import this footer component in another js file and tried using it as a tag , like <footer/>
But its not working.

Can I know how to use the swizzled component after swizzling?

**Comment by slorber** [comment, marked as answer]
The footer component is a part of the `@theme/Layout` component.

If you want to display it conditionally, you can make it render `null` inside the Footer, or not render it at all by swizzling the `@theme/Layout` component instead

> Now how to proceed, I tried to import this footer component in another js file and tried using it as a tag , like `<footer>` But its not working.

What is not working exactly: please create a repro.

You are not necessarily supposed to "use" swizzled components. They are already "used" by Docusaurus itself, and you mostly need to customize them.

**Reply by Karan-S-dev** [reply]
This worked Thanks
